app/assets/v2/js/pages/process_bounty.js

```diff
@@ -257,7 +267,7 @@ window.onload = function() {
         };
         // just sent payout
         var send_payout = function() {
-          bounty.acceptFulfillment(bountyId, fulfillmentId, {gasPrice: web3.toHex($('#gasPrice').val() * Math.pow(10, 9))}, final_callback);
+          bounty.acceptFulfillment(bountyId, fulfillmentId, {from: account, gasPrice: web3.toHex($('#gasPrice').val() * Math.pow(10, 9))}, final_callback);
```

Comment:
I had to explicitly add the `from` key because it seems to be required in the Fortmatic provider.